.swiftpm/xcode/xcshareddata/xcschemes/UINotifications.xcscheme

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Here as well, we were required to use a test plan to make testing work in Xcode 15.